lang/py3/setup.cfg

```diff
@@ -0,0 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[mypy]
+ignore_missing_imports = true
```

Comment:
Why is this?

Reply:
`mypy` tries to import all modules it encounters -- it either tries to import the typeshed library for that module, or the module itself, either way trying to get just the type annotations. But many libraries don't have such annotations, yet, so it fails to import them. This line causes it to ignore those errors.